site/.env.local

```diff
@@ -0,0 +1,2 @@
+MOSAIC_ACTIVE_MODE_URL=http://localhost:8080
```

Comment:
when run locally using `yarn serve` Mosaic will pickup changes to the original source docs

This means you do not need to edit snapshot files, which are designed to be immutable.